tests/test_set_namelist_ingest.py

```diff
@@ -0,0 +1,28 @@
+# pylint: disable=all
```

Comment:
Please do not use disable=all for pylint. Tests are meant to be covered with pylint since they are part of our codebase.

Reply:
Ok, removed, but had to add pylint: disable=unused-variable for GitHub actions